docs/workforce-management-framework/topics/con-workflow-step.adoc

```diff
@@ -0,0 +1,63 @@
+= Building your first Workflow Step/Reusing template
+
+A basic knowledge of Angular.js is requried to build a Workflow Step.
+
+== Step Structure
+Steps are divided into to linked catergories the `module` and the `definition`.
+
+=== Module
+Modules can contain one or more angular directives for rendering steps in the portal and mobile applications. 
+{WFM-RC-NameShort} steps use Angular.js directives to build user interface as part of workflows.
+Each step is divided into two types of directive:
```

Comment:
directives

Reply:
Each step is divided into two directives. "Types of directive" is misleading.